packages/protocol/contracts/L1/verifiers/SgxVerifier.sol

```diff
@@ -59,6 +60,17 @@ contract SgxVerifier is EssentialContract, IVerifier {
     error SGX_INVALID_INSTANCES();
     error SGX_INVALID_PROOF();
 
+    /// @dev Modifier that ensures the caller is either taikoL1 or SgxAndZkVerifier.
+    modifier onlyTaikoOrCombinedVerifier() {
+        if (
+            msg.sender != resolve("taiko", true)
+                && msg.sender != resolve("tier_sgx_and_pse_zkevm", true)
```

Comment:
```suggestion
                || msg.sender != resolve("tier_sgx_and_pse_zkevm", true)
```
Do you mean match one of these two addresses?

Reply:
Yes, i mean. So good as is.

IF NOT taiko AND NOT combined verifier either -> revert.